tracing/src/main/java/org/jboss/resteasy/tracing/api/providers/TextBasedRESTEasyTracingInfo.java

```diff
@@ -55,6 +55,8 @@ public boolean supports(RESTEasyTracingInfoFormat format) {
         for (int i = 0; i < messages.length; i++) {
             final RESTEasyTracingMessage message = messageList.get(i);
             final StringBuilder text = new StringBuilder();
+            // requestId
+            text.append(message.getRequestId() + ' ');
```

Comment:
Please replace with

    text.append(message.getRequestId()).append(" ");

to avoid creating an additional stringbuilder for the string concatenation

Reply:
yes sir!